### PR TITLE
[EXPLORER_OLD] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: modules/rosapps/applications/explorer-old/explorer-ro.rc [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     ReactOS Old Explorer
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
```
Comment (binarymaster) [inline modules/rosapps/applications/explorer-old/explorer-ro.rc:5]:
```suggestion
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Romanian resource file
 * TRANSLATOR:  Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
```

File: modules/rosapps/applications/explorer-old/explorer-ro.rc [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     ReactOS Old Explorer
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATOR:  Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
```
Comment (binarymaster) [inline modules/rosapps/applications/explorer-old/explorer-ro.rc:5]:
```suggestion
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Romanian resource file
 * TRANSLATORS: Copyright 2009 Petru Dimitriu <petrimetri@gmail.com>
                Copyright 2011-2014 Ștefan Fulea <stefan.fulea@mail.com>
                Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
```

File: modules/rosapps/applications/explorer-old/explorer-ro.rc [hunk 3]
```diff
@@ -1,3 +1,12 @@
+/*
+ * PROJECT:     ReactOS Old Explorer
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2009 Petru Dimitriu <petrimetri@gmail.com>
+                Copyright 2011-2014 Ștefan Fulea <stefan.fulea@mail.com>
+                Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
```
Comment (binarymaster) [inline modules/rosapps/applications/explorer-old/explorer-ro.rc:7]:
```suggestion
 * TRANSLATORS: Copyright 2009 Petru Dimitriu <petrimetri@gmail.com>
 *              Copyright 2011-2014 Ștefan Fulea <stefan.fulea@mail.com>
 *              Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
```

**Comment by AndreiMiloiu** [comment]
Thank you @binarymaster!

**Pull request merged**